### ISSUE TITLE
Removal of Handwriting (all languages) BETA causes Sysprep Error

### ISSUE THREAD

**Issue by TechBrain64** [cschneegans/unattend-generator#129]
@cschneegans, this is the other issue I mentioned.

Hypervisor: Proxmox 8.3.0
Virtio: 0.1.266
VM: Windows 11 24H2

After deploying Windows 11 24H2 using the unattend.xml file where I've checked option to remove "Handwriting (all languages) BETA"; once deployment completed I tried running Sysprep which failed with error.

I've attached the log file:
[Unattend Sysprep_Error.txt](https://github.com/user-attachments/files/18031636/Unattend.Sysprep_Error.txt)

Workaround: uncheck "Handwriting (all languages) BETA" and all is right with the world. :-)

**Comment by Hammerfest** [comment]
The key line in sysprep, and is a thorn in anyones side using sysprep is:

2024-11-27 20:46:59, Error                 SYSPRP Package Microsoft.Ink.Handwriting.Main.en-US.1.0.1_0.645.1237.0_x64__8wekyb3d8bbwe **was installed for a user, but not provisioned for all users.** This package will not function properly in the sysprep image.

Bolded the actual error cause of sysprep failure
Now, as to WHY a supposedly removed package is not fully/properly removed with the option, thats a good question I have not an answer to. All I know, is I fought with that error when using Notepad++ installing before sysprep due to Windows 11's new context menu requiring user land packages and not working with system packages... microsoft -_-

**Comment by TechBrain64** [comment]
Yes, I saw that text as well. The only thing I can think of was I installed updates and made small changes; which possibly caused the Handwriting components to install only for my account. Perhaps I can try running the uninstall before Sysprep. Is there a way to block the install permanently?

**Comment by Hammerfest** [comment]
My main recommendation is not to let it have internet until AFTER sysprep if it all possible if it is an update (as usual, microsoft installing things because they bundle it with an update) 

To solve the problem you must run the command to uninstall the user package then WITHOUT restarting (IIRC), run sysprep, otherwise for NP++, the package gets reinstalled on restart or right clicking for the user account in use.

**Comment by TechBrain64** [comment]
@Hammerfest,

I've searched for "uupdump" which yields many results. Which is your recommended project?

**Comment by Hammerfest** [comment]
https://uupdump.net/

Its been down a few times so people have done offshoots, the downloader is on github somewhere (dont recall at the moment)
When you select a build you can then select language, versions, etc etc

You can view the batch/script files as well so you can see everything it does.
